packages/atomic/src/components/atomic-result-v1/atomic-result-common.pcss

```diff
@@ -62,4 +62,18 @@
       display: none;
     }
   }
+
+  &.image-large,
+  &.image-small {
+    atomic-result-section-visual {
+      background-color: var(--atomic-neutral);
+      border-radius: 1rem;
+    }
+  }
+
+  &.image-icon {
+    atomic-result-section-visual {
+      border-radius: 0.25rem;
```

Comment:
Should the border radius adapt to the root font size? or should it be fixed t a certain curvature?

For example, someone might increase the font size in their browser for accessibility reasons. Does it make sense to have the pictures become more circular in this situation?

Reply:
Pretty much everything resizes with font-size, including image sizes. I think border radius should increase proportionally with the image size for the same reason that somebody may not be able to see small details in text or images.